src/components/VideoPlayer/Player.js

```diff
@@ -302,7 +337,9 @@ class VideoPlayer extends Component {
     const { playing, progress, muted, subtitles, loading, fullscreen, isFullscreen } = this.state
 
     return (
-      <div {...merge(styles.wrapper, breakoutStyles[size])}>
+      <div {...merge(styles.wrapper, breakoutStyles[size])}
+        onClick={this.captureFocus.bind(this)}
```

Comment:
We tend to avoid bind() when possible. If this doesn't work
`onClick={this.captureFocus}`
could you try
`onClick={() =>{ this.captureFocus()}}`

Reply:
I made the requested change. But can you elaborate or point me to a document on why bind should be avoided?

Reply:
Binding is ok, when not happening in an update cycle. Binding in the render method will create and pass a new function every time an update is happening. So if you want to bind, do it in the contructor:
```
constructor(props) {
   super(props)
   this.captureFocus = this.captureFocus.bind(this)
   // or
   this.captureFocus = () => { /* your business logic*/ }
}
```

Reply:
This however is also true for any anonymous arrow function, so the argument is somewhat tainted anyway, because that we do all the time and all over the place.

Reply:
Alright, I defined the function in the constructor.